README.md

```diff
@@ -135,14 +149,14 @@ conda deactivate
 . envs/set_vars.sh $conda_prefix
 ```
 
-Activate the environment and install the packages all the R dependencies in R directly or use the script `install_R_methods.R`.
+Activate the R environment you plan on using and call the script as follows, with the correct R dependency
+file for your environment (see table above).
 
-```commandline
-conda activate <r-environment>
-Rscript envs/install_R_methods.R
 ```
-
-For the installation of`Conos`, please see [the Conos github repo](https://github.com/hms-dbmi/conos).
```

Comment:
is conos now included in all environments and doesn't need a special install?

Reply:
Yes, conos is contained in the installation script, just like Harmony etc